packages/metamask-module-template/README.md

```diff
@@ -0,0 +1,15 @@
+# `@metamask/metamask-module-template`
```

Comment:
All files within `metamask-module-template` were added using the `create-package` script as an example package to test that all package scripts work. We can remove this one as soon as we have another package.